frontend/src/metabase/metadata/components/FkTargetPicker/FkTargetPicker.tsx

```diff
@@ -97,12 +97,17 @@ export const FkTargetPicker = ({
             <Icon name={selected ? "check" : "empty"} />
 
             <Flex direction="column" flex="1" gap="xs">
-              <Text c="inherit" lh="1rem">
+              <Text c="inherit" component="span" lh="1rem">
                 {item.option.label}
               </Text>
 
               {field?.description && (
-                <Text c="text-tertiary" className={S.description} lh="1rem">
+                <Text
```

Comment:
!nit the label/description looks a bit crammed, can we double check with design that this is what they want? I'm finding some other pages where we have tight padding (upload settings is one), but this seems inconsistent with most of our other form inputs.

![CleanShot 2025-05-05 at 16 59 09@2x](https://github.com/user-attachments/assets/2bf9ca06-d4ac-4cce-8f95-3b5e99769d43)

Also, this isn't a merge blocking comment. Feel free to circle back around / follow up in a separate PR as needed.

Reply:
Actually we have a design for that! Addressed in #57638

![image](https://github.com/user-attachments/assets/4972ab6c-a73c-4f2f-986e-d1ff7db905e0)